lib/std/os/bits/linux.zig

```diff
@@ -2244,3 +2244,8 @@ pub const MADV_COLD = 20;
 pub const MADV_PAGEOUT = 21;
 pub const MADV_HWPOISON = 100;
 pub const MADV_SOFT_OFFLINE = 101;
+
+pub const __kernel_timespec = extern struct {
```

Comment:
we usually don't include the `__` prefix when specifying these in zig.

Reply:
Yeah I wasn't sure about the name so I used what's in the pdf, good to know.